src/site/content/en/blog/share-like-a-native/index.md

```diff
@@ -0,0 +1,147 @@
+  })
+    .then(() => console.log('Successful share'))
+    .catch((error) => console.log('Error sharing', error));
+}
+```
+
+If your site has multiple URLs for the same content, share the page's
+canonical URL instead of the current URL. Instead of sharing
+`document.location.href`, you would check for a canonical URL `<meta>` tag in
+the page's `<head>` and share that. This will provide a better experience to the
+user. Not only does it avoid redirects, but it also ensures that a shared URL serves
+the correct user experience for a particular client. For example, if a friend
+shares a mobile URL and you look at it on a desktop computer,
+you should see a desktop version:
+
+```js
+let url = document.location.href;
+const canonicalElement = document.querySelector('link[rel=canonical]');
+if (canonicalElement !== null) {
+    url = canonicalElement.href;
+}
+navigator.share({url: url});
+```
+
+### Sharing files
+
+To share files, first test for and call `navigator.canShare()`. Then include an
+array of files in the call to `navigator.share()`:
+
+```js/0-4
+if (navigator.canShare && navigator.canShare({ files: filesArray })) {
+  navigator.share({
+    files: filesArray,
+    title: 'Vacation Pictures',
+    text: 'Photos from September 27 to October 14.',
+  })
+  .then(() => console.log('Share was successful.'))
+  .catch((error) => console.log('Sharing failed', error));
+} else {
+  console.log(`Your system doesn't support sharing files.`);
+}
+```
+
+Notice that the sample handles feature detection by testing for
+`naviagator.canShare()` rather than for `navigator.share()`.
+The data object passed to `canShare()` only supports the `files` property.
+Image, video, audio, and text files can be shared. (See
+[Permitted File Extensions in Chromium](https://docs.google.com/document/d/1tKPkHA5nnJtmh2TgqWmGSREUzXgMUFDL6yMdVZHqUsg/edit?usp=sharing).)
+More file types may be added in the future.
+
+## Santa Tracker case study
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./santa-phone.png" style="max-width: 400px;" alt="The Santa Tracker app showing a share button."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Santa Tracker share button.
+  </figcaption>
+</figure>
+
+[Santa Tracker](https://santatracker.google.com/), an open-source project, is a
+holiday tradition at Google. Every December, you can celebrate the season
+with games and educational experiences.
+
+In 2016, the Santa Tracker team used the Web Share API on Android.
+This API was a perfect fit for mobile
+In previous years, the team disabled share buttons on mobile because space is
+at a premium, and they couldn't justify having several share targets.
+
+But with the Web Share API, they were able to present just one button,
+saving precious pixels.
+They also found that users shared with Web Share around 20% more than
+users without the API enabled. Head to
+[Santa Tracker](https://santatracker.google.com/) to see Web Share in action.
```

Comment:
Non-blocking, but the narrative might be clearer if it went:
- What Santa Tracker is (fine as-is)
- What problem they had (had to omit share buttons on mobile)
- How Web Share API solved the problem